app/src/main/java/com/sunasterisk/music_72/screen/fragment/playtrack/PlayTrackFragment.kt

```diff
@@ -39,6 +46,18 @@ class PlayTrackFragment : Fragment(){
         initData()
     }
 
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        mediaManager = MediaManager.getInstance(context!!)
+        initMediaData()
+    }
+
+    private fun initMediaData() {
+        track?.let { mediaManager.setCurrentTrack(it) }
+        mediaManager.setTracks(tracks)
+        track?.let { mediaManager.change(it) }
```

Comment:
track?.let {
mediaManager.setCurrentTrack(it)
mediaManager.setTracks(it)
mediaManager.change(it)
}